src/transports/mod.rs

```diff
@@ -15,6 +15,9 @@ pub mod ws;
 #[cfg(any(feature = "ws-tokio", feature = "ws-async-std"))]
 pub use self::ws::WebSocket;
 
+pub mod ipc;
```

Comment:
Could you feature gate this with `ipc-tokio` and make sure that features are correctly configured in `Cargo.toml` file?

Also please add a test to `.github/workflows/rust.yml` to make sure we test the transport in isolation.

Reply:
The changes to the `.github/workflows/rust.yml` file are in 583d81638ac431cc75d7b77b05578171c061ffb5.

The feature gate was amended to the first commit.